src/server/models/index.js

```diff
@@ -95,10 +95,11 @@ function dropTables() {
   });
 }
 
+const isSqlite = /sqlite/.test(thinky.k.client.config.client);
```

Comment:
Nice -- @lastminutediorama if you decide to deprecate SQLite searching around for the `isSqlite` variable as defined here should be a good start for cleaning out tests in particular